### PR TITLE
Bug 1822845 update manual image pruning documentation 4.4-onwards

### PR DESCRIPTION
**Pull request by bmcelvee**
Related to https://github.com/openshift/openshift-docs/pull/24144.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@xltian would you mind reviewing this PR, please? Thank you!

**Comment by xltian** [comment]
@wzheng1 can you review from QE side?

**Comment by wzheng1** [comment]
LGTM, thanks!

File: modules/pruning-images-manual.adoc [hunk 1]
```diff
@@ -32,15 +103,17 @@ that have metadata in the cache will not be pushed. Therefore, you must redeploy
 ----
 $ oc rollout restart deployment/image-registry -n openshift-image-registry
 ----
-
-If the integrated registry uses a Redis cache, you must clean the database
-manually.
-
-`oc adm prune images` operations require a route for your registry. Registry
-routes are not created by default.
++
```
Comment (mburke5678) [inline modules/pruning-images-manual.adoc:106]: I assume you fixed the `+` error that currently appears in the netlify preview?

File: modules/pruning-images-manual.adoc [hunk 2]
```diff
@@ -5,20 +5,91 @@
...
-The Pruning Custom Resource enables automatic image pruning. However, administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits by running the following command:
+The Pruning Custom Resource enables automatic image pruning. However, administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits. There are two methods to manually prune images:
 
-[source,terminal]
+* Running image pruning as a `Job` or `CronJob` on the cluster.
+* Running the `oc adm prune images [<options>]` command.
+
+.Prerequisites
+
+* To manually prune images, you must first log in to the CLI as a user with an access token. The user must also have the cluster role of `system:image-pruner` or greater (for example, `cluster-admin`).
```
Comment (mburke5678) [inline modules/pruning-images-manual.adoc:15]: Do we need To manually prune images, here?
Seems a bit confusing to me. Sounds like you need this role for one of the two methods, which isn't true(?).

File: modules/pruning-images-manual.adoc [hunk 3]
```diff
@@ -5,20 +5,91 @@
...
+* Expose the image registry.
+
+.Procedure
+
+Running image pruning as a `Job` or `CronJob` on the cluster.
+
+* To manually prune images that are no longer required by the system due to age, status, or exceed limits create a YAML file for the `pruner` service account, for example:
```
Comment (mburke5678) [inline modules/pruning-images-manual.adoc:22]: Do we need _To manually prune images that are no longer required by the system due to age, status, or exceed limits_?
Just: _Create a YAML file for the `pruner` service account, for example:_ ?
Do you need to `oc create` the YAML?

**Comment by mburke5678** [comment]
@bmcelvee If you apply the structural changes in https://github.com/openshift/openshift-docs/pull/24144, make the same changes here?

File: modules/pruning-images-manual.adoc (hunk 1 above)
Comment (bmcelvee) [reply modules/pruning-images-manual.adoc:106]: I'll double check - thanks!

**Comment by bmcelvee** [comment]
> @bmcelvee If you apply the structural changes in #24144, make the same changes here?

Will do, thank you for the review!

**Comment by bmcelvee** [comment]
4.4 is giving me an error. Will manually CP after lunch.

**Pull request merged**